app/api/routes/recommendations.py

```
@@ -40,3 +47,102 @@ def get_recommendations(
     )
 
     return RecommendationResponse(**result)
+
+
+@router.get("/logs", response_model=RecommendationLogListResponse)
+def get_all_recommendation_logs(
+    page: int = Query(1, ge=1, description="페이지 번호"),
+    page_size: int = Query(20, ge=1, le=100, description="페이지 크기"),
+    db_mongo: Database = Depends(get_mongo_db),
```

Ruff (0.14.6)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)